assignments/objects.js

```diff
@@ -10,46 +10,117 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
+
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
 
+const Intern = function(id, firstName, email, gender){
+  this.id = id,
+  this.firstName = firstName,
+  this.email = email,
+  this.gender = gender
+}
+
+// 1,mmelloy0@psu.edu,Mitzi,F
+// 2,kdiben1@tinypic.com,Kennan,M
+// 3,kmummery2@wikimedia.org,Keven,M
+// 4,gmartinson3@illinois.edu,Gannie,M
+// 5,adaine5@samsung.com,Antonietta,F
 
-// ==== Challenge 2: Reading Object Data ==== 
+const internMitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F",
+};
+
+
+const internKennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function speak(){
+    return ("Hi may name is Kennan");
```

Comment:
on line 49 you don't need the `speak()` after the function, you can just create the function as `function()` and this is basically creating an anonymous function.

in the string `"Hi my name is Kennen"` could be written with a `this.name` replacing Kennen. `this` key word in this situation points to its self and retrives the value of the `name` property. an example of this could would like this

```speak: function() {return `Hi my name is ${this.name}`}```